addons/l10n_mx/models/account_account.py

```diff
@@ -1,4 +1,1090 @@
-from odoo import api, Command, models
+import re
+from odoo import _, api, Command, models
+from odoo.exceptions import RedirectWarning
+
+SAT_CODES = [
```

Comment:
if absolutely necessary (which at first glance they seem to be - condolences) , I'd move those value in a separate file in a `data` folder.

Reply:
Could this file do? https://github.com/odoo/odoo/blob/16.0/addons/l10n_mx/data/account.group.template.csv